syncopy/specest/freqanalysis.py

```diff
@@ -833,6 +870,11 @@ def freqanalysis(data, method='mtmfft', output='pow',
     # Sanitize output and call the ComputationalRoutine
     # -------------------------------------------------
 
+    # If provided, make sure output object is appropriate
+    if out is not None:
```

Comment:
This should not be needed anymore, as after the notorious `cfg` merge the support for `out` got removed